engine-tests/src/test/java/org/terasology/documentation/ModuleMarkupScraper.java

```diff
@@ -0,0 +1,444 @@
+            Module latestVersion = moduleManager.getRegistry().getLatestModuleVersion(moduleId);
+            if (!latestVersion.isOnClasspath()) {
+                gameModules.add(latestVersion);
+                System.out.println("Found Game Module: " + latestVersion.getId());
+            }
+        }
+
+        List<Module> remoteModules = Lists.newArrayList();
+        Iterable<Module> remoteModuleRegistry = moduleManager.getInstallManager().getRemoteRegistry();
+        Set<Name> filtered = ImmutableSet.of(TerasologyConstants.ENGINE_MODULE, new Name("engine-test"));
+        for (Module remote : remoteModuleRegistry) {
+            remoteModules.add(remote);
+            System.out.println("Found Remote Module: " + remote.getId());
+        }
+        allModules.addAll(gameModules);
+        allModules.addAll(remoteModules);
+        allModules.sort(moduleInfoComparator);
+
+        List<Module> allSortedModules;
+        allSortedModules = new ArrayList<>(allModules);
+
+        StringBuilder wiki = new StringBuilder();
+
+        StringBuilder moduleMapListing = new StringBuilder();
+
+        List<Module> sortedGameModules = new ArrayList<>(gameModules);
+        sortedGameModules.sort(moduleInfoComparator);
+        List<Module> sortedRemoteModules = new ArrayList<>(remoteModules);
+        sortedRemoteModules.sort(moduleInfoComparator);
+        moduleMapListing.append("| ");
+        for (Module gModule : sortedGameModules) {
+            // Add anchor tags, for example [AdditionalFruits](#AdditionalFruits)
+            moduleMapListing.append("[").append(gModule.getId()).append("](#").append(gModule.getId()).append(") ");
+        }
+        moduleMapListing.append(" | ");
+        for (Module rModule : sortedRemoteModules) {
+            moduleMapListing.append("[").append(rModule.getId()).append("](#").append(rModule.getId()).append(") ");
+        }
+        moduleMapListing.append(" | ");
+
+        StringBuilder moduleMap = new StringBuilder();
+        moduleMap.append(System.getProperty("line.separator"));
+        moduleMap.append("**Extensions:**");
+        moduleMap.append(System.getProperty("line.separator"));
+        moduleMap.append("| Installed | Remote |");
+        moduleMap.append(System.getProperty("line.separator"));
+        moduleMap.append("| -------- | ---------- |");
+        moduleMap.append(System.getProperty("line.separator"));
+        moduleMap.append(moduleMapListing);
+
+        wiki.append(moduleMap);
+
+        wiki.append(ScanModules(allSortedModules));
+        System.out.println(wiki);
+
+        String fileName = "Module_Wiki.md";
+        BufferedWriter writer = new BufferedWriter(new FileWriter(new File(fileName)));
+        writer.write(wiki.toString());
+        writer.flush();
+        writer.close();
+        System.out.println("Module Wiki generation file is ready!");
+
+        env.close();
+    }
+
+    private static String ScanModules(List<Module> allSortedModules) {
```

Comment:
This is a method and thus should start with a lower case letter:
```suggestion
    private static String scanModules(List<Module> allSortedModules) {
```

Reply:
applied lowerCamelCase